client/blocks/eligibility-warnings/warning-list.tsx

```diff
@@ -15,14 +15,18 @@ interface ExternalProps {
 type Props = ExternalProps & LocalizeProps;
 
 export const WarningList = ( { context, translate, warnings, showContact = true }: Props ) => {
-	const hasEnTranslation = useHasEnTranslation();
 	return (
 		<div>
-			{ getWarningDescription( context, warnings.length, translate, hasEnTranslation ) && (
+			{ getWarningDescription( context, warnings.length, translate ) && (
 				<div className="eligibility-warnings__warning">
 					<div className="eligibility-warnings__message">
-						<span className="eligibility-warnings__message-description">
-							{ getWarningDescription( context, warnings.length, translate, hasEnTranslation ) }
+						<span
+							className={ `eligibility-warnings__message-description ${
+								context === 'hosting-features' &&
+								'eligibility-warnings__message-description--hosting-features'
```

Comment:
I added a custom class to change the font weight here. Maybe unnecessary if we're okay with the font weight changing elsewhere.